### ISSUE TITLE
Allow contact addresses in Address Book to be copied to clipboard

### ISSUE THREAD

**Issue by ethgasstation** [safe-global/safe-ios#3336]
# Safe mobile ios requires that user scan a QR code or paste an address from the clipboard into the 'Recipient's address' field when initiating a send transaction.   Ideally, the app would also allow a user to link to an address in the contacts list stored in the 'Address Book' when initiating a send transaction but at minimum user should be able copy to clipboard the address from the address book and then paste into the "Recipient Address' field.  Currently, the user cannot copy to clipboard from the address book. 

As a user, I would like to copy and paste (or link directly to) an address from the address book to the 'Recipient's address' field in the send transaction, so that i can complete the transaction to a known address easily and securely.  

# What To Do

[Technical steps to do]
1.   Provide a copy button to the contract addresses in the Address Book. 
2.   Provide option to link directly to address book in the 'Recipient Address' field of send transactions 
3. 

# Acceptance Criteria

* [Testable and user-facing criteria]
* 
* 

# Resources

* Links
* 
*

**Comment by TobiWo** [comment]
I agree to 100%. It should be possible to choose an address from your address book directly.

In detail: When you want to send assets and you tap on the `Recipient's address` field you should get the following options:
* Paste from Clipboard
* Scan QR Code
* Select from Address Book

Thanks a ton for this awesome app.
